Regression/WarpX-tests.ini

```
@@ -3824,7 +3824,7 @@ analysisRoutine = Examples/Tests/point_of_contact_EB/analysis.py
 [ThetaImplicitPicard_1d]
 buildDir = .
 inputFile = Examples/Tests/Implicit/inputs_1d
-runtime_params = warpx.abort_on_warning_threshold=high
+runtime_params = warpx.abort_on_warning_threshold=high amr.max_grid_size=20
```

`development` was:
```suggestion
runtime_params = warpx.abort_on_warning_threshold=high amr.max_grid_size=32
```
Just picked `20` because grid size is `40`.